client/my-sites/stats/site.jsx

```diff
@@ -110,56 +107,19 @@ module.exports = React.createClass( {
 	},
 
 	render: function() {
-		var site = this.props.sites.getSite( this.props.siteId ),
-			charts = this.props.charts(),
-			queryDate = this.props.date.format( 'YYYY-MM-DD' ),
-			oldStatsLink = 'https://wordpress.com/my-stats/?poll=1&blog=' + this.props.siteId,
-			currentUser = user.get(),
-			period = this.props.period.period,
-			moduleStrings = statsStrings(),
-			oldStatsMessage = null,
-			tagsList,
-			nonPeriodicModules,
-			videoList;
+		const site = this.props.sites.getSite( this.props.siteId );
+		const charts = this.props.charts();
+		const queryDate = this.props.date.format( 'YYYY-MM-DD' );
+		const period = this.props.period.period;
+		const moduleStrings = statsStrings();
+		let nonPeriodicModules;
+		let videoList;
 
 		debug( 'Rendering site stats component', this.props );
 
-		if ( ! site.jetpack && 'wp-admin' === store.get( 'oldStatsLink' ) ) {
```

Comment:
I don't know how we handled the transition from old to new stats, so I'm merely pointing out that some functionality seems to be removed here.